corehq/apps/hqwebapp/urls.py

```diff
@@ -3,8 +3,9 @@
 
 urlpatterns = patterns(
     'corehq.apps.hqwebapp.views',
+    (r'^$', 'redirect_to_default'),
     url(r'^homepage/$', 'redirect_to_default', name='homepage'),
-    url(r'^home/$', 'landing_page', name='landing_page'),
+    url(r'^default_landing/$', 'landing_page', name='landing_page'),
```

Comment:
What will happen to people who go to `/home/` once this is in effect?

Personally I think `http://commcarehq.org/home/` is prettier than `http://commcarehq.org/default_landing/`. Will anyone ever see this URL?